boards/arm/frdm_kw41z/dts.fixup

```diff
@@ -1,3 +1,7 @@
 #define CONFIG_FXOS8700_NAME			NXP_KINETIS_I2C_40067000_NXP_FXOS8700_1F_LABEL
 #define CONFIG_FXOS8700_I2C_NAME		NXP_KINETIS_I2C_40067000_NXP_FXOS8700_1F_BUS_NAME
 #define CONFIG_FXOS8700_I2C_ADDRESS		NXP_KINETIS_I2C_40067000_NXP_FXOS8700_1F_BASE_ADDRESS
+#define CONFIG_RTC_MCUX_0_IRQ_PRI		NXP_KINETIS_RTC_4003D000_IRQ_0_PRIORITY
+#define CONFIG_RTC_MCUX_0_IRQ			NXP_KINETIS_RTC_4003D000_IRQ_0
+#define CONFIG_RTC_MCUX_0_NAME			NXP_KINETIS_RTC_4003D000_LABEL
+#define CONFIG_RTC_PRESCALER			NXP_KINETIS_RTC_4003D000_PRESCALER
```

Comment:
These changes should go in the soc-level fixup instead

Reply:
Fixed in https://github.com/zephyrproject-rtos/zephyr/pull/7182/commits/ec1a2be6d82313da7d5f761e8fda07c7d71808f6